README.md

```diff
@@ -81,69 +47,46 @@ every 5 seconds.
+The Po.et Node comes with a default configuration that works out of the box, which can be found here:
+https://github.com/poetapp/node/blob/master/src/Configuration.ts#L82-L141
 
-You can change any configuration by placing a JSON file in `~/.po.et/configuration.json`. Po.et will look for this file upon startup and, if found, merge its contents with the default configuration.
+By default, anchoring to the blockchain is disabled (`enableTimestamping: false`). If you want to enable blockchain anchoring, you will need to the Bitcoin Core dependency to be running with a funded wallet so that it can pay the Bitcoin network transaction fees (either for testnet or real Bitcoin for mainnet).
 
-Alternatively, you can pass configuration values via environment variables. The keys of these environment variables are always the SCREAMING_SNAKE_CASE equivalent of the configuration options listed in the default configuration. For example, the RabbitMQ URL can be set with a `rabbitmqUrl` entry in `~/.po.et/configuration.json` or with the `RABBITMQ_URL` environment variable.
+You can change any configuration by passing configuration values via environment variables. The keys of these environment variables are always the SCREAMING_SNAKE_CASE equivalent of the configuration options listed in the default configuration. For example, the RabbitMQ URL (`rabbitmqUrl`) can be set with the `RABBITMQ_URL` environment variable.
 
 > **Note**: Po.et will NOT reload the configuration while it's running if you change it. You will need to restart the Node for configuration changes to apply.
 
-This is what the default configuration looks like:
-
-```js
-{
-  rabbitmqUrl: 'amqp://localhost',
-  mongodbUrl: 'mongodb://localhost:27017/poet',
-  ipfsUrl: 'http://localhost:5001',
-  ipfsArchiveUrlPrefix: 'https://ipfs.io/ipfs',
-  bitcoinUrl: '127.0.0.1',
-  bitcoinPort: 18332,
-  bitcoinNetwork: 'testnet',
-  bitcoinUsername: 'bitcoinrpcuser',
-  bitcoinPassword: 'bitcoinrpcpassword',
-
-  apiPort: 18080,
-  poetNetwork: 'BARD',
-  poetVersion: [0, 3],
-  minimumBlockHeight: 1253828,
-  blockchainReaderIntervalInSeconds: 5,
-
-  enableTimestamping: false,
-  anchorIntervalInSeconds: 30,
-  batchCreationIntervalInSeconds: 600
-  readNextDirectoryIntervalInSeconds: 30
-}
-```
+## API
+Currently, the Node exposes four endpoints.
 
-The node works out-of-the-box with the default configuration as long as it's running with the MongoDB, IPFS, RabbitMQ and Bitcoin Core docker images provided in this repository (see [docker-compose.yml](./docker-compose.yml)).
+### `GET /works?issuer=xxx&limit=x&offset=x`
+Returns a paginated array of signed verifiable claims, all belonging to the passed issuer (if an issuer is passed). Default limit per request is 10 claims. This is configurable with limit and offset paramaters where offset is the number of claims to skip and limit is the number of claims returned per request.
```

Comment:
I'm thinking of changing this to something like:

Returns a paginated array of signed verifiable work claims.

Accepts the following query parameters:
- `issuer`: string. If present, will only return works issued by this issuer.
- `limit`: number. Maximum number of results to return per request. Defaults to 10.
- `offset`: number. Number of claims to skip.

What do you think?

Reply:
Much better. I've updated it.